<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>withdrawal</class>
 <widget class="QDialog" name="withdrawal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(0, 0, 0)</string>
  </property>
  <widget class="QPushButton" name="withdrawalButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>withdrawal</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="withdrawalLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#depoAmountLabel {
	font-size: 18px
}</string>
   </property>
   <property name="text">
    <string>withdrawal amount:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="withdrawalValueText">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
